.github/workflows/lint-tests-release.yml

```diff
@@ -63,10 +63,26 @@ jobs:
         working-directory: ./contract-examples
       - name: Run e2e tests
         shell: bash
-        run: E2E=true scripts/run.sh
+        run: SKIP_NETWORK_RUNNER_START=true SKIP_NETWORK_RUNNER_SHUTDOWN=true ENABLE_SOLIDITY_TESTS=true scripts/run.sh
```

Comment:
Do we need two different variables here `SKIP_NETWORK_RUNNER_START` and `SKIP_NETWORK_RUNNER_SHUTDOWN` ?